src/overlays/actors/ovl_En_Hidden_Nuts/z_en_hidden_nuts.h

```diff
@@ -8,7 +8,10 @@ struct EnHiddenNuts;
 typedef void (*EnHiddenNutsActionFunc)(struct EnHiddenNuts*, PlayState*);
 
 #define ENHIDDENNUTS_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
-#define ENHIDDENNUTS_GET_F80(thisx) (((thisx)->params >> 7) & 0x1F)
+#define ENHIDDENNUTS_GET_PATH_INDEX(thisx) (((thisx)->params >> 7) & 0x1F)
+
+#define ENHIDDENNUTS_PATH_INDEX_NONE 0x1F
+#define ENHIDDENNUTS_PATH_INDEX_NONE_ALT 0x3F
```

Comment:
man I knew it was different for different actors... but even in the same actor?

Reply:
Yeah, looks like a copy paste error. `ENHIDDENNUTS_PATH_INDEX_NONE` should be the real one, but use the `ALT` version in the SubS function. However it is harmless since they have the check just before, but probably should have used `0x1F` again.